src/lara-app/hooks/interactive-api.ts

```diff
@@ -39,6 +39,15 @@ export interface IFirebaseJWT {
   };
 }
 
+// TODO: move this interface to lara-interactive-api?
```

Comment:
Right, it should be either in LARA Interactive API or in question-interactives, but not really here. Currently, there's a second copy in question-interactives that is used by the graph interactive.

It doesn't actually make 100% sense to keep it in LARA Interactives API, as it's actually mostly bound to graph interactive. But that way, we wouldn't have to publish another package from question-interactives just to export this interface. And it could be reused by other interactives later.

Reply:
Yeah I think it makes sense to move it to lara-interactive-api.

Another thing in the interface is that it should probably support `null` in the cells. It is common to have sparse data.

I'm pretty sure we'll have to evolve this as we support other kinds of datasets.  But the version number should help with that, and if the only consumer of the data is the graph interactive, it shouldn't be too hard to support new versions.